doc/source/whatsnew/v1.3.0.rst

```diff
@@ -366,8 +366,8 @@ ExtensionArray
 Other
 ^^^^^
 - Bug in :class:`Index` constructor sometimes silently ignorning a specified ``dtype`` (:issue:`38879`)
--
--
+- Bug in :func:`pandas.api.types.infer_dtype` not recognizing Series, Index or array with a period dtype (:issue:`23553`)
+- Bug in :func:`pandas.api.types.infer_dtype` raising an error for general :class:`.ExtensionArray` objects. It will now return ``"unknown-array"`` instead of raising (:issue:`37367`)
```

Comment:
This last entry might not really fit in the bug fixes section .. but maybe also not worth adding a dedicated subsection for it in the "notable (API breaking) bug fixes"